cli/main.go

```diff
@@ -26,13 +27,33 @@ func execute() error {
 	return cmd.ExecuteContext(ctx)
 }
 
-var version = "0.0.0-dev"
+var (
+	version          = "0.0.0-dev"
+	runtimeHandler   = "contrast-cc"
+	launchDigest     = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+	genpolicyVersion = "0.0.0-dev"
+)
 
 func newRootCmd() *cobra.Command {
+	// build the versions string
+	var versionsBuilder strings.Builder
+	versionsBuilder.WriteString(fmt.Sprintf("%s:\n\n", version))
+	versionsBuilder.WriteString(fmt.Sprintf("\truntime handler: %s\n", runtimeHandler))
+	versionsBuilder.WriteString(fmt.Sprintf("\tlaunch digest: %s\n", launchDigest))
+	versionsBuilder.WriteString(fmt.Sprintf("\tgenpolicy version: %s\n", genpolicyVersion))
+	versionsBuilder.WriteString("\timage versions:\n")
+	imageReplacements := strings.Trim(string(cmd.ReleaseImageReplacements), "\n")
+	for _, image := range strings.Split(imageReplacements, "\n") {
+		if !strings.HasPrefix(image, "#") {
+			image = strings.Split(image, "=")[1]
+		}
+		versionsBuilder.WriteString(fmt.Sprintf("\t\t%s\n", image))
```

Comment:
```suggestion
		if !strings.HasPrefix(image, "#") {
			image = strings.Split(image, "=")[1]
			versionsBuilder.WriteString(fmt.Sprintf("\t\t%s\n", image))
		}
```

That way, we don't get the placeholder text from the file in dev builds.